contracts/protocol/bases/GroupBase.sol

```diff
@@ -120,7 +121,8 @@ contract GroupBase is ProtocolBase, IBosonGroupEvents {
      * @return valid - validity of condition
      *
      */
-    function validateCondition(Condition memory _condition) internal pure returns (bool valid) {
+    function validateCondition(Condition memory _condition) internal pure returns (bool) {
```

Comment:
```suggestion
    function validateCondition(Condition calldata _condition) internal pure returns (bool) {
```
I just noticed this.
My last request, I promise :blush:

Reply:
done :)